src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -67,7 +67,7 @@ public function catalogAction()
             'PrestaShopBundle:Admin/Module:catalog.html.twig',
             [
                 'layoutHeaderToolbarBtn' => $this->getToolbarButtons(),
-                'layoutTitle' => $this->trans('Module selection', 'Admin.Navigation.Menu'),
+                'layoutTitle' => $this->trans('Modules catalog', 'Admin.Navigation.Menu'),
```

Comment:
Should it be `Module catalog`?

Reply:
I kept the previous title from AddonsStoreController (look here https://github.com/PrestaShop/PrestaShop/pull/10200/commits/a4811350682fe0e654f68428fba1cf6141023d69#diff-42da191bd5eec313d41734188486e993R52)